src/Components/People.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchPeople: "",
+      currentPeople: {},
```

Comment:
"people" is probably not the word you want for these `searchPeople` and `currentPeople` variables. There's never going to be more than one person here, so `searchPerson` and `currentPerson` are probably better names. Otherwise you (or someone else) can easily mistake it for an array of people.